src/Components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,91 @@
+// Themes
+@import '../../constants/Themes/MainTheme';
+
+.tooltipContainer {
+  position: relative;
+  display: inline-block;
+  color: white;
+
+  .tooltipMsg {
+    background: $Color-TooltipBG;
+    border-radius: 5px;
+    border: solid 1px darkgray;
```

Comment:
You already set a var for the bg color. It whould be awesome if you do the same fot the border.

Reply:
In fact would be awesome. Done.